hyprpm/src/main.cpp

```diff
@@ -11,7 +11,7 @@
 
 const std::string HELP = R"#(┏ hyprpm, a Hyprland Plugin Manager
 ┃
-┣ add [url]              → Install a new plugin repository from git
+┣ add [url] [git rev]    → Install a new plugin repository from git
```

Comment:
mention its optional, please

Reply:
Should I use `<>` to signify optional vars? Or just mention it in the msg?

Reply:
up to you. If you do <> you need to mention that it means optional ,tho